### DISCUSSION TITLE
Problem when i try using Subscribe feature

### DISCUSSION THREAD

**Discussion by lokize** [prisma/prisma#17851] [Q&A]
Hi friends, when i try using $Subscribe feature i have this error: 

TypeError: Cannot read properties of undefined (reading "order")

Strange because the "table" [order] exists.

**Comment by nurul3101** [comment]
Hello @lokize 👋 
Can you please elaborate on which $subscribe feature are you talking about?
Can you share the schema file and the query that you are invoking?

**Reply by lokize** [reply]
thanks for reply. of course.

i try using example of $subscribe in documentation.
![image](https://user-images.githubusercontent.com/19701630/218095376-3543f5b1-9422-4b62-991b-ee2922adae9d.png)

my prisma schema is:

![image](https://user-images.githubusercontent.com/19701630/218095717-7e40cedb-40f2-4dbe-be61-97f672521417.png)

when i try using $subscribe i got the error:

![image](https://user-images.githubusercontent.com/19701630/218096593-142e9206-5b9b-4ee8-abf8-b5ba9dd35ace.png)

**Reply by nurul3101** [reply]
It looks like you are referring to Prisma 1 that is no longer maintained and have been sunset. What is your use case that you are trying to achieve?

**Reply by lokize** [reply]
ah yes I was really confused about the version and documentation, I'm using "@prisma/client": "^4.7.1" can you tell me how to use subscribe in this version?

**Reply by janpio** [reply]
That is currently not supported as we operate on a lower, database level. You can follow this issue for updates: https://github.com/prisma/prisma/issues/298

**Reply by lokize** [reply]
ok thanks for help!

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
